flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -249,7 +249,15 @@ if [ -n "$HADOOP_HOME" ]; then
     fi
 fi
 
-INTERNAL_HADOOP_CLASSPATHS="$HADOOP_CLASSPATH:$HADOOP_CONF_DIR:$YARN_CONF_DIR"
+INTERNAL_HADOOP_CLASSPATHS="${HADOOP_CLASSPATH}:${HADOOP_CONF_DIR}:${YARN_CONF_DIR}"
+
+if [ -n "${HBASE_CONF_DIR}" ]; then
+    # Setup the HBase classpath.
+    INTERNAL_HADOOP_CLASSPATHS="${INTERNAL_HADOOP_CLASSPATHS}:`hbase classpath`"
```

Comment:
what happens if the call to `hbase` fails, for example because its not in the `$PATH` ?